### PR TITLE
Allow disbursements with zero points

### PR DESCRIPTION
**Pull request by kxmbrian**
As a staff member, I want to indicate to a student that marks have been already been disbursed to the class, but s/he got none.

Tutors must now take note to leave the 'points awarded' field blank if s/he does not intend to award points to a student.

### CODE REVIEW

File: app/models/course/experience_points/disbursement.rb [hunk 1]
```diff
@@ -69,9 +69,7 @@ def save
   # @param [Hash] attributes Experience points record attributes hash
   # @return [Boolean] True if hash represents a valid points award
   def valid_points_record_attributes?(attibutes)
-    attibutes[:course_user_id].present? &&
-      attibutes[:points_awarded].present? &&
-      attibutes[:points_awarded].to_i != 0
```
Comment (weiqingtoh) [inline app/models/course/experience_points/disbursement.rb]: Shall we retain the validation that `experience_points` should be `>= 0` instead? 

Although I did remember there was some discussions about negative experience points, although I don't think we will support that use case now? Or do we?

Comment (allenwq) [inline app/models/course/experience_points/disbursement.rb]: Quite weird to me to have negative EXP ><

Comment (kxmbrian) [inline app/models/course/experience_points/disbursement.rb]: Fixed!

File: spec/features/course/experience_points/disbursement_spec.rb [hunk 2]
```diff
@@ -61,10 +61,9 @@
         find(content_tag_selector(student_to_award_points)).find('input.points_awarded').set '100'
         find(content_tag_selector(student_to_set_zero)).find('input.points_awarded').set '00'
 
-        # ExperiencePointsRecord is not created when points_awarded is zero
         expect do
           click_button I18n.t('course.experience_points.disbursement.new.submit')
-        end.to change(Course::ExperiencePointsRecord, :count).by(1)
+        end.to change(Course::ExperiencePointsRecord, :count).by(2)
```
Comment (weiqingtoh) [inline spec/features/course/experience_points/disbursement_spec.rb]: Shall we also test the boundary case where experience_points_awarded is negative, and not expected the change the `:count` here?

**Comment by fonglh** [comment]
Negative exp could be used for penalties I guess.
